### DISCUSSION TITLE
cannot drag blocks using rc-dock

### DISCUSSION THREAD

**Discussion by badiuciprian** [GrapesJS/grapesjs#4893] [Q&A] [answered]
Hi, Im using grapesjs inside a rc-dock and the blocks dragging is not working. On hover the blocks respond but they cannot be grabbed. have you guys encountered this issue? I know it might be an issue with the dock but I can't seem to find a solution for this. Thanks!

**Comment by artf** [comment, marked as answer]
Try to disable the native D&D
```js
grapesjs.init({
  // ...
  nativeDnD: false,
})
```

**Reply by badiuciprian** [reply]
Great, thank you! It works but there is still a small issue, when I drag components on the artboard, they are automatically placed on top left corner. I can move and resize them, but the original position is top left. Before disabling the native dnd they were placed on the cursor location.

**Reply by artf** [reply]
Are you using absolute positioning?

**Reply by badiuciprian** [reply]
yes

**Reply by artf** [reply]
The absolute dragMode is in kind of maintenance mode https://github.com/GrapesJS/grapesjs/issues/3770 😅

**Reply by badiuciprian** [reply]
:) tough job to please everybody. Indeed I guess is more complex, but I have the feeling that this absolute dragmode is used more and more and is what people need for a long time :) So I will stick to it as it is for the moment, and maybe in the near future will be fixed. Thanks a lot for your help!
